benchmarks/benchmark-ab.py

```diff
@@ -550,9 +550,24 @@ def extract_entity(data, pattern, index, delim=" "):
     for line in data:
         if pattern.search(line):
             return line.split(delim)[index].strip()
+
+    # If code execution reaches here, there is some error
+    with open(execution_params["metric_log"]) as f:
+        data = f.readlines()
+    extract_errors(data, "Error")
+
     return None
 
 
+def extract_errors(data, pattern):
+
+    pattern = re.compile(pattern)
+    for line in data:
+        if pattern.search(line):
+            print(line)
+    assert ()
```

Comment:
nit but I prefer to use a `raise` for errors instead of manual asserts like this, I presume you want the code to just stop at this point and give a human readable error?

Reply:
Yes, thats right. Sounds good. Will change it to `raise`